sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -97,6 +109,12 @@ struct _pi_platform {
   // Current number of L0 Command Lists created on this platform.
   // this number must not exceed ZeMaxCommandListCache.
   std::atomic<int> ZeGlobalCommandListCount{0};
+
+  // Keep track of all contexts in the platform. This is needed to manage
+  // a lifetime of memory allocations in each context when there are kernels
+  // with indirect access.
+  std::list<pi_context> Contexts;
```

Comment:
Again, please add TODO that this should be deleted when memory isolation in the driver is done.

Reply:
Fixed.